source/Symbol/SwiftASTContext.cpp

```diff
@@ -6234,7 +6234,7 @@ VisitNodeQualifiedArchetype (SwiftASTContext *ast,
     {
         swift::Demangle::Node::iterator end = cur_node->end();
         VisitNodeResult type_result;
-        size_t index = LLDB_INVALID_ADDRESS;
+        uint64_t index = LLDB_INVALID_ADDRESS;
```

Comment:
This should be "lldb::addr_t" instead of "uint64_t" if we are initializing this with LLDB_INVALID_ADDRESS.